apps/web/screens/ServiceDetails/ServiceDetails.tsx

```diff
@@ -4,27 +4,94 @@ import { withMainLayout } from '@island.is/web/layouts/main'
 import getConfig from 'next/config'
 import { CustomNextError } from '@island.is/web/units/errors'
 
-const { publicRuntimeConfig } = getConfig()
+import { GetNamespaceQuery } from '@island.is/web/graphql/schema'
+import {
+  Query,
+  QueryGetApiServiceByIdArgs,
+  QueryGetNamespaceArgs,
+} from '@island.is/api/schema'
```

Comment:
This should be imported from the web's schema at `@island.is/web/graphql/schema` generated with codegen